src/FlickrAPI.js

```diff
@@ -48,7 +48,7 @@ module.exports = (function Flickr() {
         signature = Utils.sign(data, options.secret, options.access_token_secret),
         flickrURL = url + "?" + queryString + "&oauth_signature=" + signature;
 
-    request.get(flickrURL, function(error, response, body) {
+    request.defaults(options.requestOptions).get(flickrURL, function(error, response, body) {
```

Comment:
this code implies that `options.requestOptions` are way wider than just being for setting the request timeout. We should either have users supply the timeout as dedicated option and then pass that in here, using
```
request.defaults({ timeout: options.requestTimeout }).get(....)
```
or we should document that the `options.requestOptions` can be used to set any and all values that are accepted by the `request.defaults()` function